src/Hint/All.hs

```diff
@@ -38,29 +38,30 @@ data HintBuiltin =
     HintList | HintListRec | HintMonad | HintLambda |
     HintBracket | HintNaming | HintPattern | HintImport | HintExport |
     HintPragma | HintExtensions | HintUnsafe | HintDuplicate | HintRestrict |
-    HintComment | HintNewType | HintSmell
+    HintComment | HintNewType | HintSmell | HintNewTypeDerivingStrategies
```

Comment:
I actually mean merge it even further so they are just under `HintNewtype` - no entry here at all.

Reply:
:ok_hand: